### DISCUSSION TITLE
ブロック引用とコードブロックが接触している

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#356] [要望・アイデア・不具合]
ブロック引用内にコードブロックを配置すると、コードブロックの黒色背景がブロック引用の灰色垂直棒に接触しています。

重大な問題ではありませんが、もう少し離した方が見やすいように感じます。

![image](https://user-images.githubusercontent.com/75878492/201602133-87520f8d-6b7c-4bd4-8b4e-15197364d5af.png)

<details><summary>サンプルコード</summary>

```markdown
> ```python
> print("Hello World")
> ```

-----

>     print("Hello World")
```

</details>

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご報告ありがとうございます！
こちらも #354 と合わせて調査の上、検討いたします🙇

**Comment by zonuexe** [comment]
左側の接触もそうですが、前後の余白が間延びしすぎて視認性が悪化して困っています。

特にこのコードブロック引用は本文でも使いますが特にコメントで頻繁に使っています。

-----

Markdownではありませんが、Slackのように間延びしすぎない程度が理想的です。

<img width="265" alt="スクリーンショット 2022-11-19 18 27 28" src="https://user-images.githubusercontent.com/822086/202844360-e5012d4c-a4d5-480f-8991-346c08b5746d.png">

**Reply by tanaka-asami415** [reply]
@zonuexe 
ご報告ありがとうございます🙏こちらも合わせて検討します！

**Comment by Phroneris** [comment]
不自然な余白の無さなので是非改善されてほしいです

**Comment by kimkim0814** [comment]
@Yarakashi-Kikohshi @zonuexe @Phroneris
お待たせして申し訳ありません。

ブロック引用とコードブロックの間に適切な余白を挿入し、
コードブロックの上下の余白が間延びしないように調整しました😊

| before  | after |
| ------------- | ------------- |
|![変更前](https://github.com/increments/qiita-discussions/assets/54876760/687073d1-eb16-4b91-8cfa-28fb66dc8127)|![変更後](https://github.com/increments/qiita-discussions/assets/54876760/274705fe-c97c-449b-a1ae-29c5b87a8f5f)|

**Reply by yarakos95** [reply]
@kimkim0814 

実際にブロック引用内のコードブロックが灰色の垂直棒に接触していない様子を確認しました 👀 
調整ありがとうございます！ 🎉
